catalog-rest-service/src/main/resources/json/schema/entity/services/connections/dashboard/modeConnection.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$id": "https://open-metadata.org/schema/entity/services/connections/dashboard/modeConnection.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "ModeConnection",
+  "description": "Mode Connection Config",
+  "type": "object",
+  "javaType": "org.openmetadata.catalog.services.connections.dashboard.ModeConnection",
+  "definitions": {
+    "modeType": {
+      "description": "Mode service type",
+      "type": "string",
+      "enum": ["Mode"],
+      "default": "Mode"
+    }
+  },
+  "properties": {
+    "type": {
+      "title": "Service Type",
+      "description": "Service Type",
+      "$ref": "#/definitions/modeType",
+      "default": "Mode"
+    },
+    "access_token": {
```

Comment:
could you please use camelCase in the JSON Schemas? Feels a bit weird when translated to python but follows the codebase convention 🙏

Reply:
oh yes...I'll update